### ISSUE TITLE
How doable would it be to get sys/mman.h functions?

### ISSUE THREAD

**Issue by coolreader18** [ndless-nspire/Ndless#174]
We're missing `mmap` and `munmap`, would it be feasible to get those working with the ndless SDK?

**Comment by Vogtinator** [comment]
Nope, the OS doesn't do virtual memory management. It would be possible to implement `mmap` for allocating memory in Ndless itself, but that's about it.
